.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "es5",
+  "printWidth": 120,
+  "singleQuote": true
```

Comment:
I would rather go with singleQuotes false and semicolin true

Reply:
Airbnb one the style guides actually recommends using single quotes

Reply:
main thing in this pr was to setup it, we need to add more rules, then we can talk about it, and agree which to choose